src/components/form/attributes/CustomAttributes.tsx

```diff
@@ -67,8 +74,10 @@ function CustomAttribute({ attribute, index }: CustomAttributeProps) {
 
 export function CustomAttributesSection({
     schemaSection,
+    useSectionedLayout = false,
```

Comment:
I would prefer not to use 'use' as it has strong connotations with hooks. 
```suggestion
    sectionLayout = false,
    //or 
    isSectionLayout = false,
```